ops/docker/Dockerfile.packages

```diff
@@ -3,7 +3,7 @@
 #
 
 # Stage 0 (named `manifests`) collects
-# dependency manifest files (`package.json` and `yarn.lock`) which are then
+# dependency manifest files (`package.json` and `pnpm-lock.yaml`) which are then
```

Comment:
I won't do it in my pr but we can get rid of this docker multi-stage hack I added because pnpm supports doing docker monorepos via copying only pnpm-lock file https://pnpm.io/cli/fetch